app/javascript/src/components/Dashboard/Notes/NewNotePane.jsx

```diff
@@ -1,16 +1,140 @@
+          description: yup.string().required("Description is required"),
+          role: yup
+            .object({
+              label: yup.string(),
+              value: yup.string()
+            })
+            .required("Role required"),
+          tags: yup.array().min(1).required("Tag required")
+        })}
+      >
+        {({ isSubmitting }) => (
+          <Form className="w-full">
+            <Pane.Body>
+              <Input
+                label="Title"
+                name="title"
+                className="mb-6"
+                placeholder="Enter Title"
+                required={true}
+              />
+              <Input
+                label="Description"
+                name="description"
+                className="mb-6"
+                placeholder="Enter Description"
+                size={"large"}
+                required={true}
+              />
+              <div className="mb-6">
+                <Select
+                  isClearable
+                  isSearchable
+                  required={true}
+                  label="Assigned Contact"
+                  name="role"
+                  options={[
+                    {
+                      label: "Value One",
+                      value: "value1"
+                    },
+                    {
+                      label: "Value Two",
+                      value: "value2"
+                    },
+                    {
+                      label: "Value Three",
+                      value: "value3"
+                    },
+                    {
+                      label: "Value Four",
+                      value: "value4"
+                    },
+                    {
+                      label: "Value Five",
+                      value: "value5"
+                    }
+                  ]}
+                  placeholder="Select a Role"
+                />
+              </div>
+              <div className="mb-6">
+                <Select
+                  isClearable
+                  isSearchable
+                  required={true}
+                  isMulti
+                  label="Tags"
+                  name="tags"
+                  options={[
+                    {
+                      label: "Getting Started",
+                      value: "Getting Started"
+                    },
+                    {
+                      label: "Onboarding",
+                      value: "Onboarding"
+                    },
+                    {
+                      label: "User Flow",
+                      value: "User Flow"
+                    },
+                    {
+                      label: "UX",
+                      value: "UX"
+                    },
+                    {
+                      label: "Bugs",
+                      value: "Bugs"
+                    }
+                  ]}
+                  placeholder="Select Tags"
+                />
+              </div>
```

Comment:
When you have multiple elements and you wish to space them evenly, it's better to use [space between](https://tailwindcss.com/docs/space#class-reference) utils from tailwindCSS rather than defining spacing utils for each element.